arkouda/dataframe.py

```diff
@@ -1475,12 +1475,21 @@ def to_hdf(self, path, index=False, columns=None, file_type="distribute"):
         file_type: str (single | distribute)
             Default: distribute
             Whether to save to a single file or distribute across Locales
-
+        Returns
+        -------
+        None
+        Raises
+        ------
+        RuntimeError
+            Raised if a server-side error is thrown saving the pdarray
         Notes
         -----
         This method saves one file per locale of the arkouda server. All
         files are prefixed by the path argument and suffixed by their
         locale number.
+        See Also
+        ---------
+        to_hdf, load
```

Comment:
This `See Also` section should probably read `to_parquet, load`

Reply:
Updated